mordecai3/geoparse.py

```diff
@@ -29,8 +29,12 @@
 
 spacy_doc_setup()
 
+spacy_model_name = os.environ.get("SPACY_MODEL_NAME", "en_core_web_trf")
+mordecai_model_pt = "assets/mordecai_2024-06-04.pt"
```

Comment:
do you know what is this file for?

Reply:
No idea, looks like some kinda model they have .. very small but it looks like a model